CMakeLists.txt

```diff
@@ -28,5 +28,8 @@ file(GLOB SOURCE_FILES "src/*.cpp" "src/*.h")
 # Declare target executable
 add_executable(${TARGET} ${SOURCE_FILES})
 
+# Link libraries
+target_link_libraries(${TARGET} template-library)
```

Comment:
Thanks for the observation @rpannek. The issue has been addressed in the commit below.